intake/src/main/scala/VariantProcessingStage.scala

```diff
@@ -13,7 +13,8 @@ class VariantProcessingStage(implicit context: Context) extends Stage {
     applications = Seq.empty,
     masterVolumeSizeInGB = 100,
     bootstrapScripts = Seq(new BootstrapScript(resourceUri("processor_bootstrap.sh"))),
-    instances = 1
+    instances = 1,
+    releaseLabel = ReleaseLabel("emr-6.7.0")
```

Comment:
Makes the bootstrap go from 20 minutes to 4 minutes. I believe this is because it can install a more recent version of scipy which was the bottleneck before. They must have fixed something.

Reply:
I see that you added a ReleaseLabel where one was not present previously.  Does it default to `ReleaseLabel.emrDefault`?  It also looks like `ReleaseLabel.emrLatest` is behind the 6.7 referenced here.